lib/views/account.html

```diff
@@ -32,6 +32,7 @@ <h3>Account Management</h3>
           <ul class="nav nav-tabs">
             <li class="active"><a href="#email" data-toggle="tab">Email Address</a></li>
             <li><a href="#password" data-toggle="tab">Password</a></li>
+            <li><a href="#jobsQuantityOnPage" data-toggle="tab">Jobs quantity on page</a></li>
```

Comment:
The `enabled` flag is stored in the strider config and you can’t simply `require` it because the only getter in `config.js` returns the whole object. I tried to create a `virtual` model field, but for some reason this field is not available at the client side.
Also I can’t figure out how to get current user object in `ProjectsController`. In other controllers this object is already available in `global` scope, but it's not true for `ProjectsController`.